EDU-17772 - Add guide for VTEX CX Platform's MCP (#2608)
vtexdocs/dev-portal-content: docs/release-notes/2026-04-15-vtex-cx-platform-mcp-connection-guide.md

---

> title: "VTEX CX Platform: Connect the MCP to your AI tool"

Reviewer: ```suggestion
title: "VTEX CX Platform: New MCP server for AI-powered customer support insights"
```

slug: "2026-04-15-vtex-cx-platform-mcp-connection-guide"
type: "added"
createdAt: "2026-04-15T13:00:00.000Z"
updatedAt: "2026-04-15T13:00:00.000Z"

> excerpt: "Connect the VTEX CX Platform MCP to AI tools such as Claude, Cursor, and ChatGPT."

Reviewer: ```suggestion
excerpt: "Use the VTEX CX Platform MCP server to connect customer support analytics to AI tools such as Claude, Cursor, and ChatGPT"
```

Reviewer: acho importante reforçar que é sobre customer support data - até para não confundir com o VTEX Developer MCP

---

> Now, the VTEX CX Platform offers a [Model Context Protocol (MCP) server](https://developers.vtex.com/docs/guides/connect-the-vtex-cx-platform-mcp) that can connect your data in the VTEX CX Platform to an AI tool.

Reviewer: ```suggestion
The **VTEX CX Platform (Weni)** now provides a [Model Context Protocol (MCP) server](https://developers.vtex.com/docs/guides/connect-the-vtex-cx-platform-mcp)that allows AI tools to securely access customer support analytics.
```

> MCP provides as a standardized and secure way for AI assistants to access your **VTEX CX Platform** data, so you can query and analyze customer support information directly from your preferred AI tool.

Reviewer: ```suggestion
With the **VTEX CX Platform MCP** server, AI assistants can securely access **VTEX CX Platform** data based on your permissions, allowing you to query interactions, analyze conversations, and generate insights from customer support analytics directly from your preferred AI tool.
```

Reviewer: achei melhor ser repetitiva com "VTEX CX Platform MCP" para não confundir com o VTEX Developer MCP

## What has changed?

> With the VTEX CX Platform MCP, developers and support teams can:

Reviewer: ```suggestion
With the **VTEX CX Platform MCP**, developers and support teams can:
```

> - Explore customer support data.

Reviewer: ```suggestion
- Access customer support analytics from AI tools.
```

> - Run interaction analyses.

Reviewer: ```suggestion
- Analyze customer interactions and conversations.
```

Reviewer: > Run interaction analyses.

Bem genérico. Interações dos clientes?

> - Generate insights about interactions with customers.

Reviewer: Interações do suporte com os clientes?

## What needs to be done?

> To connect the VTEX CX Platform MCP to your AI tool, follow the instructions in the [Connect the VTEX CX Platform MCP](https://developers.vtex.com/docs/guides/connect-the-vtex-cx-platform-mcp) guide.

Reviewer: ```suggestion
To start using the **VTEX CX Platform MCP**, follow the instructions in the [Connect the VTEX CX Platform MCP](https://developers.vtex.com/docs/guides/connect-the-vtex-cx-platform-mcp) guide.

For details on how your data is handled when using the **VTEX CX Platform MCP** server, see:

- [Terms of use](https://mcp.weni.ai/terms)
- [Privacy policy](https://mcp.weni.ai/privacy)
```

---
title: "VTEX CX Platform: New MCP server for AI-powered customer support insights"
slug: "2026-04-15-vtex-cx-platform-mcp-connection-guide"
type: "added"
createdAt: "2026-04-15T13:00:00.000Z"
updatedAt: "2026-04-15T13:00:00.000Z"
excerpt: "Use the VTEX CX Platform MCP server to connect customer support analytics to AI tools such as Claude, Cursor, and ChatGPT"
---

> The **VTEX CX Platform (Weni)** now provides a [Model Context Protocol (MCP) server](https://developers.vtex.com/docs/guides/connect-the-vtex-cx-platform-mcp)that allows AI tools to securely access customer support analytics.

Reviewer: ```suggestion
The **VTEX CX Platform (Weni)** now provides a [Model Context Protocol (MCP) server](https://developers.vtex.com/docs/guides/connect-the-vtex-cx-platform-mcp) that allows AI tools to access and analyze your customer support analytics.
```
